thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range/template.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="col-xs-4">
+  {{date-range-picker
```

Comment:
no more labels for the input fields here?

Reply:
left them out for now for vspace. may return soon